frontend/src/utils/pagination.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from "react";
+import { useBetterPaginatedQuery } from "../services/apiService";
+import { Pagination as BoostrapPagination } from "react-bootstrap";
+
+export const Pagination = ({ render, apiKey, apiMethod, apiParams }) => {
+    let [page, setPage] = useState(1);
+    let [maxPage, setMaxPage] = useState(1);
+    const { resolvedData: data, status, error } = useBetterPaginatedQuery(
+        apiKey,
+        apiMethod,
+        ...apiParams,
+        page
```

Comment:
So, is it how it works: when `page` changes thanks to the callback sent to `PaginationControl`, `useBetterPaginatedQuery` which delegates to the effect `usePaginatedQuery`… updates `data`?